src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -2203,20 +2192,20 @@
 (defn compile-user-query->sql
   "Given a user provided query and a Query instance, convert the
    user provided query to SQL and extract the parameters, to be used
-   in a prepared statement"
+   in a prepared statement. Every query coming into this is a 'from' clause."
   [query-rec user-query & [{:keys [include_total] :as paging-options}]]
   ;; Call the query-rec so we can evaluate query-rec functions
   ;; which depend on the db connection type
-  (let [allowed-fields (map keyword (queryable-fields query-rec))
+  (let [entity (keyword (second user-query))
         paging-options (some->> paging-options
-                                (paging/validate-order-by! allowed-fields)
                                 (paging/dealias-order-by query-rec))
         {:keys [plan params]} (->> user-query
                                    (push-down-context query-rec)
                                    expand-user-query
-                                   (convert-to-plan query-rec paging-options)
+                                   (convert-to-plan paging-options)
                                    extract-all-params)
-        sql (-> plan fix-plan-in-expr-multi-comparisons plan->sql)
+        plan (assoc plan :wrap? true)
```

Comment:
yes, but previously the decision to wrap was based on whether a query was a subquery. Since the subquery concept is gone now, this is just telling the engine to always wrap the top level query (which enters here) in the inactive nodes CTE. Behavior should be the same as before.